presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -40,7 +40,13 @@ public TestMySqlDistributedQueries()
 
     public TestMySqlDistributedQueries(TestingMySqlServer mysqlServer)
     {
-        super(() -> createMySqlQueryRunner(mysqlServer, ImmutableMap.of(), TpchTable.getTables()));
+        super(() -> createMySqlQueryRunner(
+                mysqlServer,
+                ImmutableMap.<String, String>builder()
+                        .put("metadata.cache-ttl", "10m")
```

Comment:
add a comment why you enable caching here and where are the test without caching